### PR TITLE
Separate links for mmsi/imo:s

### PR DESCRIPTION
**Pull request by redhog**
Connects https://github.com/SkyTruth/pelagos-server/issues/829

### CODE REVIEW

**Comment by enriquetuya** [comment]
@redhog you need to remove the link from the Vessel Info header.
https://github.com/SkyTruth/pelagos-client/blob/2fba1414dead9b65d868e6d95f778f2b16090780/js/app/Visualization/UI/BasicSidebar.js#L215

File: js/app/Visualization/UI/BasicSidebar.js [hunk 1]
```diff
@@ -176,8 +176,39 @@ define([
             self.node.find(".vessel_id .flag").html("---");
           }
 
-          self.node.find(".vessel_id .imo").html(data.imo || "---");
-          self.node.find(".vessel_id .mmsi").html(data.mmsi || "---");
+          if (data.imo) {
```
Comment (enriquetuya) [inline js/app/Visualization/UI/BasicSidebar.js]: @redhog Both imo and mmsi code are basically the same. Why don't you encapsulate them?

display_with_links(data.imo, ".vessel_id .imo", "link")

``` javascript
display_with_link(values, css_style, the_url){
          if (values) {
             self.node.find(css_style).html("");
             var first = true;
             values.split(",").map(function (value) {
               var link = $("<a target='_blank'>");
               link.text(mmsi);
               link.attr({href: the_url + value});
               if (!first) {
                 self.node.find(css_style).append(", ");
               }
               self.node.find(css_style).append(link);
               first = false;
             });
           } else {
             self.node.find(css_style).html("---");
  }
}
```

Comment (enriquetuya) [inline js/app/Visualization/UI/BasicSidebar.js]: ping @redhog

Comment (enriquetuya) [inline js/app/Visualization/UI/BasicSidebar.js]: ping @redhog x2

Comment (redhog) [inline js/app/Visualization/UI/BasicSidebar.js]: Only because there's so little code that it wouldn't make any difference.

Comment (enriquetuya) [inline js/app/Visualization/UI/BasicSidebar.js]: If you sum up little code that wouldn't make difference you always end up having lots of code that end up on difficult code to maintain.

**Pull request merged**